src/evaluate/evaluator/base.py

```diff
@@ -216,7 +219,7 @@ def compute(
 
         return result
 
-    def prepare_data(self, data: Union[str, Dataset], input_column: str, label_column: str):
+    def prepare_data(self, data: Union[str, Dataset], input_column: str, label_column: str, **kwargs):
```

Comment:
I think we should keep the signature explicit:
```suggestion
    def prepare_data(self, data: Union[str, Dataset], input_column: str, label_column: str, data_split: str = None):
```

Reply:
Not a fan of "mysterious hidden box of kwargs which could hide absolutely anything inside them and possibly break your entire function", got it 😉